README.md

```diff
@@ -90,6 +90,9 @@ Finally, to run the api locally:
 npm run dev
 ```
 
+- View the documentation locally via `localhost:3000`
```

Comment:
Could we reorder this a little? It's clearer to me to first make a point that the application is running on port `3000`. After that first point, we can make these sub points. sub-point 1, the root route `/` has the documentation built from the point where the application was run. sub-point 2, the `/api/{endpoint}` routes have the actual implementations